applications/nrf_desktop/src/modules/hid_state.c

```diff
@@ -608,6 +613,51 @@ static void send_report_mouse(void)
 	}
 }
 
+static void send_report_consumer_ctrl(void)
+{
+	if (IS_ENABLED(CONFIG_DESKTOP_HID_CONSUMER_CTRL)) {
+		struct report_data *rd =
+			&state.report_data[IN_REPORT_CONSUMER_CTRL];
+		struct hid_consumer_ctrl_event *event =
+			new_hid_consumer_ctrl_event();
+
+		event->subscriber = state.selected->id;
+		event->bitmask = 0;
+
+		/* Traverse pressed keys and build consumer control report */
+		for (size_t i = 0; i < ARRAY_SIZE(rd->items.item); i++) {
+			struct item *item = &rd->items.item[i];
+
+			if (item->value) {
```

Comment:
I know this is copy paste but this is wrong - can you please check `if (item->usage_id)` instead and assert value?

Reply:
ok